android/src/main/java/io/parity/signer/bottomsheets/PrivateKeyExportBottomSheet.kt

```diff
@@ -0,0 +1,122 @@
+package io.parity.signer.bottomsheets
+
+import android.content.res.Configuration.UI_MODE_NIGHT_NO
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.core.graphics.drawable.toBitmap
+import io.parity.signer.R
+import io.parity.signer.components.NetworkCardModel
+import io.parity.signer.components2.KeyCard
+import io.parity.signer.components2.KeyCardModel
+import io.parity.signer.models.EmptyNavigator
+import io.parity.signer.models.Navigator
+import io.parity.signer.models.intoImageBitmap
+import io.parity.signer.ui.theme.*
+import io.parity.signer.uniffi.Action
+import io.parity.signer.uniffi.Address
+
+@Composable
+fun PrivateKeyExportBottomSheet(
+	model: PrivateKeyExportModel,
+	navigator: Navigator,
+) {
+
+//	val bottomSheetScaffoldState = rememberBottomSheetScaffoldState(
+//		bottomSheetState = BottomSheetState(BottomSheetValue.Collapsed)
+//	)
+//	val coroutineScope = rememberCoroutineScope()
+
+	Column(
+		modifier = Modifier.clickable { navigator.navigate(Action.GO_BACK)}
+			.fillMaxWidth()
+	) {
+		Spacer(Modifier.weight(1f))
+		Surface(
+			color = MaterialTheme.colors.Bg000,
+			shape = MaterialTheme.shapes.modal
+		) {
+			Column(
+				modifier = Modifier
+					.fillMaxWidth()
+			) {
+				Row(
+					Modifier
+						.padding(top = 3.dp, start = 12.dp, end = 12.dp)
```

Comment:
ok will do

Reply:
btw padding resources needed to adapt UI to different devices (eq. bigger paddings for tablets). Not to change them since it's very difficult to define proper abstractions before - if we'll change corner radius for QR codes, we most probably won't change the same for frames, buttons et.c.

Reply:
Android dimen resources are the instrument to change dimens automatically based on device qualifiers. Without this feature there would be no reason to use them at all. 
But it doesn't mean we need to define everything in resources. With compose we should reuse components and dimens specific to this component now can be defined next to the component, rather than in global namespace in xml.
Many google samples not using resource dimens anymore.
I mean there can be use of them, or there can be more downsides as with any instrument.  For now I"m trying to understand which dimens are specific to components and which meant to be shared.